skel/config/app.yml

```diff
@@ -15,3 +15,11 @@ database:
 redis:
   host: localhost
   port: 6379
+
+mngapi:
+  root_url: 
```

Comment:
I think we need the ability to use different base URL for peatio and barong, inside the cluster they are differents.
Do we really need to split the url and prefix?

Reply:
Ok, I will remove `root_url` and use `peatio_url` and `barong_url` instead.